server/admin_management/api/app/routers/router_guards.py

```diff
@@ -358,3 +358,31 @@ def validate_param_access_control_privilege_id(
         utils.raise_http_exception(
             error_code=ERROR_INVALID_ACCESS_CONTROL_PRIVILEGE_ID, error_msg=error_msg
         )
+
+
+def authorize_by_application_admin(
```

Comment:
Hi Jazz, the unused method still exists. Could you remove it before you resolve this comment? That might help on test coverage as well. Thanks.